index.js

```diff
@@ -1,8 +1,22 @@
 const config = {
+    env: {
+        es6: true,
+        jest: true,
+        browser: true,
+        node: true
+    },
     plugins: ['jsx-a11y', 'package-json', 'react', 'react-hooks'],
-    extends: ['plugin:jsx-a11y/recommended', 'plugin:package-json/recommended'],
+    extends: [
+        'eslint:recommended',
+        'prettier',
```

Comment:
This disables rules that conflict with `prettier`. Should we also leave this up to the consuming app/repo? I don't feel strongly about this one.

Reply:
I feel strongly that in the modern ecosystem, "formatting" code is a different step than checking it for style and good practice. ESLint has rules that cover the former and the latter, because it predates things like Prettier, but I think of those rules as "legacy" ESlint; we should limit ESLint to look at code style and not code formatting.

Reply:
Totes. All I'm asking is whether all users of this config will be using prettier and if not, would this extends still make sense?

Reply:
I hope that all users of this config would use prettier. If they don't use it, they could use an override.